src/api/public/apidocs/paths/source_project_name_package_name_view_info.yaml

```diff
@@ -10,19 +10,11 @@ get:
       name: view
       schema:
         type: string
+        # There are further view points, but this documentation is just for view=info
         enum:
-          - cpio
-          - getmultibuild
           - info
-          - issues
-          - products
-          - productrepositories
       description: |
-        Specify which information about a package should be returned.
-
-        * `info`: Show source version, md5sums and build description files, among other information.
-        * `cpio`, `getmultibuild`, `issues`, `products`, `productrepositories`:
-          See this [other endpoint](<#/Sources - Files/get_source__project_name___package_name_>) for details.
```

Comment:
Removing this link to the other endpoint will make harder for users to find the endpoint they need. Why do you think removing this section is needed?

Reply:
Same reason as given above: having a link to the endpoint with the other values the view parameter can have doesn't harm. Way the contrary, it helps users find or relate what they are looing for.